cmd/manager/main.go

```diff
@@ -321,14 +320,16 @@ func parseTargetTable(targetTableEnv string) int {
 	}
 }
 
-func parseProtectedSubnets(subnetsEnv string) []*net.IPNet {
+func collectProtectedSubnets(envVars []string) []*net.IPNet {
 	protectedSubnets := []*net.IPNet{}
-	for _, subnet := range strings.Split(subnetsEnv, ",") {
-		_, subnetNet, err := net.ParseCIDR(strings.Trim(subnet, " "))
-		if err != nil {
-			panic(err)
+	for _, e := range envVars {
+		if v := strings.SplitN(e, "=", 2); strings.Contains(v[0], "PROTECTED_SUBNET_") {
+			_, subnetNet, err := net.ParseCIDR(strings.Trim(v[1], " "))
```

Comment:
I suggest to allow multiple CIDRs in one environment variable separated by comma

Reply:
Thanks, I put back that.